### PR TITLE
Clarify integration (and potential conflicts) between the Process and its normative references

### PR DESCRIPTION
**Pull request by frivoal**
This addresses the part of @jwrosewell's formal objection on precedence and potential conflicts between the Process and its normative references.

@jwrosewell, can you review and let us know whether this change addresses your comment? (See https://github.com/w3c/w3process/pull/571 for the Pull Request that addresses the rest of your comment)

***
<a href="https://pr-preview.s3.amazonaws.com/frivoal/w3process/pull/572.html" title="Last updated on Aug 31, 2021, 5:01 AM UTC (426be81)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/572/b9d22e7...frivoal:426be81.html" title="Last updated on Aug 31, 2021, 5:01 AM UTC (426be81)">Diff</a>

***
<a href="https://pr-preview.s3.amazonaws.com/frivoal/w3process/pull/572.html" title="Last updated on Aug 3, 2023, 3:16 AM UTC (8ea7ee5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/572/2574b40...frivoal:8ea7ee5.html" title="Last updated on Aug 3, 2023, 3:16 AM UTC (8ea7ee5)">Diff</a>

### CODE REVIEW

**Comment by frivoal** [comment]
> When we eliminated "Last Call" from the process (circa 2014?), we did not want to change the PP. So in the process, we defined CR to be Last Call from the point of view of the PP. It is not clear that this would have worked if the PP had precedence.

I do not believe this to be a case of conflict between the Patent Policy and the Process. The Patent Policy depended on the term "Last Call", and the process changed which phase of the document's evolution this corresponds to.

Both documents are designed to work together, and depend on each other's terminology to make sense. Through these dependencies, we can evolve them as we did for Last Call. But if the Process accidentally stated something incorrect about the patent policy, it should not change the meaning of the patent policy.

**Comment by frivoal** [comment]
> Minor, but I would prefer "takes precedence" to "overrides"

No objection, but I am unsure what nuance this difference would imply.

> and we should have a statement that these policies are written with the intent that there be no conflict, but in the event that one exists, the MA takes precedence over etc.

Sure. Updated the text accordingly.

**Comment by jeffjaffe** [comment]
> > When we eliminated "Last Call" from the process (circa 2014?), we did not want to change the PP. So in the process, we defined CR to be Last Call from the point of view of the PP. It is not clear that this would have worked if the PP had precedence.
> 
> I do not believe this to be a case of conflict between the Patent Policy and the Process. The Patent Policy depended on the term "Last Call", and the process changed which phase of the document's evolution this corresponds to.
I would not say that I know you are wrong, my only point was that "it is not clear to me".  If PSIG signs off that there is no ambiguity, then I am OK with the past example about Last Call.

But I also worry about future changes as I said above.  Given that we never want to change the PP, but we change the Process every year, there could be slight changes that we need to make where it would be convenient to make them in the Process - but couldn't if it conflicts with the PP.

I'm also interested in understanding why it is important for the PP to have precedence over the Process.  If the documents work together, there wouldn't be any conflict.

**Comment by dwsinger** [comment]
> I'm also interested in understanding why it is important for the PP to have precedence over the Process. If the documents work together, there wouldn't be any conflict.

That's not what we are talking about. We hope, intend, and believe that we are perfect and there are and will never be conflict. This text deals with situations where we are wrong, and made a mistake.

I am pretty sure that a document that puts legal obligations on members takes precedence over something that puts procedural requirements on them, and that PSIG would agree.

**Comment by jeffjaffe** [comment]
> I am pretty sure that a document that puts legal obligations on members takes precedence over something that puts procedural requirements on them, and that PSIG would agree.

Terrific!!  @wseltzer , can we get PSIG to comment on this thread?

**Comment by wseltzer** [comment]
> > Minor, but I would prefer "takes precedence" to "overrides"
> 
> No objection, but I am unsure what nuance this difference would imply.

It's more common legal language, so more likely to be understood as "in the event of conflict, look to the one with precedence as controlling."

**Comment by frivoal** [comment]
@dwsinger @wseltzer updated the pull request to use "takes precedence" rather than "overrides"

**Comment by jwrosewell** [comment]
Thank you. The revision with "precedence" makes sense.

I'm unclear regarding the interpretation of the following sentence.

"The Process Document, Patent Policy, and CEPC have been designed to allow them to evolve independently."

My concern is knowing which one applies when interpreting the W3C Membership Agreement, Appendix 1, and these interrelated dependent documents. It seems this is being progressed in PR [571](https://github.com/w3c/w3process/pull/571) and might be removed from this PR?

**Comment by nigelmegitt** [comment]
As per https://github.com/w3c/w3process/issues/574#issuecomment-925006702 I think this change is misplaced and belongs in the referencing document, i.e. the Member Agreement, not the Process.

**Comment by fantasai** [comment]
@nigelmegitt I think this could go in either the Member Agreement or the Process, and it's a lot easier to put it in the Process, so I think it's fine here.

File: index.bs [hunk 1]
```diff
@@ -117,18 +117,24 @@ g g:hover text {
...
+	These policies are written with the intent that there be no conflict,
+	but in the event that one exists,
+	the Member Agreement takes precedence over the Patent Policy,
+	which takes precedence over this Process,
+	which takes precedence over any <a href="#normative">other normative references</a>.
+
+	The Patent Policy and CEPC place additional obligations on Members, Team, and other participants in W3C.
 	The Process Document does not restate those requirements but includes references to them.
-	The Process Document and Patent Policy have been designed to allow them to evolve independently.
+	The Process Document, Patent Policy, and CEPC have been designed to allow them to evolve independently.
```
Comment (TallTed) [inline index.bs:137]:
```suggestion
	The Process Document, Patent Policy, and CEPC have been designed to allow each to evolve independently.
```

File: index.bs [hunk 2]
```diff
@@ -109,20 +109,26 @@ g g:hover text {
...
-
-	The Patent Policy places additional obligations on Members, Team, and other participants in W3C.
+	and other policies
+	incorporated by normative reference as a part of the Process Document
+	are thus equally binding.
+	These policies are written with the intent that there be no conflict,
+	but in the event that one exists,
+	the Member Agreement takes precedence over the Patent Policy,
+	which takes precedence over this Process,
+	which takes precedence over any <a href="#normative">other normative references</a>.
```
Comment (nigelmegitt) [inline index.bs:127]: The link indicates this means "other normative references" _from this Process_ but that's a subtle way of conveying the information. Consider "other normative references from this Process" as a clearer alternative?

Comment (frivoal) [inline index.bs:127]:
```suggestion
	which takes precedence over any <a href="#normative">other normative references from this Process</a>.
```

Comment (nigelmegitt) [reply index.bs:127]: Thanks @frivoal, that would be a good suggestion to take up.

**Review by tantek — APPROVED** [review]
Good to make this explicit.

**Review by tantek — APPROVED** [review]

**Review by Alex10099 — APPROVED** [review]

File: index.bs [hunk 3]
```diff
@@ -109,20 +109,26 @@ g g:hover text {
 <!--[if lt IE 9]><script src='undefined://www.w3.org/2008/site/js/html5shiv.js'></script><![endif]-->
 
 <h2 class="no-num no-toc no-ref" id="pp">
-Relation of Process Document to Patent Policy</h2>
+Relation of Process Document to Patent Policy and Other Policies</h2>
 
 	<!-- DO NOT REMOVE THIS SECTION OR ITS REFERNCE TO THE PATENT POLICY WITHOUT DISCUSSING WITH PSIG / COUNSEL. -->
```
Comment (TallTed) [inline index.bs:114]:
```suggestion
	<!-- DO NOT REMOVE THIS SECTION OR ITS REFERENCE TO THE PATENT POLICY WITHOUT DISCUSSING WITH PSIG / COUNSEL. -->
```

**Review by Alex10099 — APPROVED** [review]

**Comment by frivoal** [comment]
As stated in https://github.com/w3c/w3process/issues/574#issuecomment-1802211225:
> action florian to review PR 572 to see if there is any editorial bit to salvage before closing

This PR had 4 commits. The [first one](https://github.com/w3c/w3process/pull/572/commits/9dc2780a3cda092ef755db78c746f2bb73921f43) and [last one](https://github.com/w3c/w3process/pull/572/commits) being editorial, and worth salvaging, but that has already been done a while back, with commit 16f6156c3afb48f1cf17caed6b81644a40727b42, on the main branch since February 2023.

The rest was substantive, but is being abandoned, as per PSIG's recommendation documented in #574. Closing

**Pull request closed without merging**